lib/stat_tracker.rb

```diff
@@ -1,41 +1,43 @@
 require "csv"
 
 class StatTracker
-  
+  attr_reader :games,
+              :teams,
+              :game_teams
+
+  def initialize
+    @games = []
+    @teams = []
+    @game_teams =[]
+  end
+
   def self.from_csv(locations)
-    game_hash = {}
     stat_tracker = StatTracker.new
     
-    game_hash[:games] = stat_tracker.game_generator(locations)
-    game_hash[:teams] = stat_tracker.teams_generator(locations)
-    game_hash[:game_teams] = stat_tracker.game_teams_generator(locations)
-    game_hash
+    stat_tracker.game_generator(locations)
+    stat_tracker.teams_generator(locations)
+    stat_tracker.game_teams_generator(locations)
+    return stat_tracker
```

Comment:
THAT'S the way to make the instance variables accessible? Interesting. Glad you all figured this out!